index.js

```diff
@@ -52,7 +59,14 @@ async function asanaOperations(
             core.info(`Task ${taskId} marked as complete.`);
         }
     } catch (error) {
-        core.setFailed(error.message);
+        if (typeof error === "string") {
+            error = new Error(error);
+        }
+        if (error instanceof Error) {
+            core.setFailed(error.message);
+        }
+
+        core.setFailed(`Unknown error: ${error}`);
```

Comment:
question: should this be in an `else` block?

Reply:
Thanks! https://github.com/mbta/github-asana-action/compare/cd12a5322d75f257321d711c3aa7073c1b9573d1..4b47fd554d1f8ec51e67a000b6bc9fcb4b000ca6